src/ring_jwt_middleware/core.clj

```diff
@@ -155,15 +155,15 @@
                        (validate-jwt jwt jwt-max-lifetime-in-sec jwt-check-fn)]
                 (log-and-refuse (pr-str jwt)
                                 (format "(%s) %s"
-                                        (:user-identifier jwt "Unkown User ID")
+                                        (:sub jwt "Unkown User ID")
                                         (str/join ", " validation-errors)))
                 (if (is-revoked-fn jwt)
                   (log-and-refuse
                    (pr-str jwt)
                    (format "JWT revoked for %s"
                            (:user-identifier jwt "Unkown User ID")))
```

Comment:
`user-identifier` is still valid at this place?

Reply:
`user-identifier` was used in our proprietary code and it should not be used anywhere.